### PR TITLE
Update help command

### LINKED ISSUE #135: Update Help command to include Transaction

### PR DESCRIPTION
**Pull request by notle1706**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/BuyCommand.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public class BuyCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a transaction and links to the company. "
             + "Parameters: "
-            + "Index "
+            + "Index (must be a positive integer) "
```
Comment (Jo3LW) [inline src/main/java/seedu/address/logic/commands/BuyCommand.java:25]: All seems good! Perhaps adding non-zero positive integer would be more informative.

File: src/main/java/seedu/address/logic/commands/SellCommand.java [hunk 2]
```diff
@@ -22,7 +22,7 @@ public class SellCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Creates a transaction and links to the company. "
             + "Parameters: "
-            + "Index "
+            + "Index (must be a positive integer) "
```
Comment (Jo3LW) [inline src/main/java/seedu/address/logic/commands/SellCommand.java:25]: All seems good! Perhaps adding non-zero positive integer would be more informative.

**Review by Jo3LW — APPROVED** [review]
LGTM

**Review by ElginL — APPROVED** [review]
LGTM!

**Pull request merged**